auth/simple_token.go

```diff
@@ -52,6 +54,21 @@ func (as *authStore) assignSimpleTokenToUser(username, token string) {
 	}
 
 	as.simpleTokens[token] = username
+	deleteTokenFunc := func() {
+		as.simpleTokensMu.Lock()
+		_, ok := as.simpleTokenLock[token]
+		if ok {
+			delete(as.simpleTokenLock, token)
+			as.simpleTokensMu.Unlock()
+			return
+		}
+		delete(as.simpleTokens, token)
+		delete(as.simpleTokenInf, token)
+		as.simpleTokensMu.Unlock()
+		plog.Errorf("deleted the token %s for user %s", token, username)
+	}
+	timer := time.AfterFunc(SimpleTokenTTL, deleteTokenFunc)
```

Comment:
time.Timer can be easier to use incorrectly
https://github.com/golang/go/issues/14038
but in this case, we are not reading the Timer.C channel and using Stop() before Reset() as recommended in the post and godoc